tests/test_25_settings_users.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+
+from selenium.webdriver.common.by import By
+
+
+@pytest.fixture(scope="module")
+def add_user_rick(admin_session, browser_session):
+    """Go to Settings User and add user rick."""
+    browser_session.select("#linkSettings").click()
+    browser_session.select("a[href='/settings/users']").click()
+
+    users = [
+        tr.text
+        for tr in browser_session.select_all("#tableUsers tr")
+    ]
+    if "rick" not in users:
```

Comment:
it's no longer needed as the scope is `module` ?

Reply:
that's right, thanks !